src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -629,8 +637,9 @@ def deploy_pods(self, warnet):
                 self.client.create_namespaced_service(
                     namespace=self.namespace, body=lightning_service
                 )
-            # add metrics scraping for tanks configured to export metrics
-            self.apply_prometheus_scrape_configs(warnet.tanks)
+
+        # add metrics scraping for tanks configured to export metrics
+        self.apply_prometheus_service_monitor(warnet.tanks)
```

Comment:
in 95610e44f38e2210d93fdffac73cc51a0ae0ed97 Add prometheus to logging stack:

There seems to be many things modified here which were only added in the previous commit. Perhaps just squash them together? Alternatively just leave; it's not worth splitting and rebasing in any case.

Reply:
squashed them together